### PR TITLE
clean .gitmodules

### PR DESCRIPTION
**Pull request by NicBOMB**
Fix 27/51 submodule definitions which used `\\` escapes to instead use a regular `/`.
The submodules have been mostly alphabetically sorted.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,201 +1,203 @@
...
+	path = extensions/common-interpreters
+	url = https://github.com/Nexus-Mods/extension-common-interpreters.git
+	branch = master
+[submodule "extensions/documentation"]
+	path = extensions/documentation
+	url = https://github.com/Nexus-Mods/extension-documentation.git
+	branch = master
+[submodule "extensions/extension-dashlet"]
+	path = extensions/extension-dashlet
+	url = https://github.com/Nexus-Mods/extension-extension-dashlet.git
```
Comment (ghost) [inline .gitmodules]: `extension-extension-dashlet` should probably be changed to just `extension-dashlet`.

File: .gitmodules [hunk 2]
```diff
@@ -1,201 +1,203 @@
...
-	url = https://github.com/Nexus-Mods/extension-changelog-dashlet
-	branch = master
-[submodule "extensions/fnis-integration"]
-	path = extensions/fnis-integration
-	url = https://github.com/Nexus-Mods/fnis-integration.git
-  branch = master
 [submodule "extensions/game-pillarsofeternity2"]
 	path = extensions/game-pillarsofeternity2
 	url = https://github.com/Nexus-Mods/game-pillarsofeternity2.git
 	branch = master
```
Comment (ghost) [inline .gitmodules]: Are there any plans to move this into vortex-games?

**Comment by TanninOne** [comment]
Sorry, this PR contains way too many unrelated changes. Also, please avoid unnecessary deltas like sorting a file. All that does is make it harder to review your changes

**Comment by TanninOne** [comment]
Thank you!

Could you explain why you want to sort the modules? If there is a technical reason you could make a new PR, if it's purely for "esthetics", honestly: just leave it. This file is primarily maintained by the git command line tool, I don't think it's a great idea to have anyone who does a "git submodule add" to then have to go into the file and fix the order, it's just another workstep that could go wrong and break the build.

**Pull request merged**